src/main/java/it/gov/pagopa/payhub/activities/exception/ValueNotValidException.java

```diff
@@ -0,0 +1,9 @@
+package it.gov.pagopa.payhub.activities.exception;
+
+
+public class ValueNotValidException extends ActivitiesException {
```

Comment:
I see we have already implemented many exceptions to represent not valid data, but there we used the form Invalid<kind of data>Exception
```suggestion
public class InvalidValueException extends ActivitiesException {
```